### ISSUE TITLE
DF won't start with Phoebus graphics 

### ISSUE THREAD

**Issue by drfragment** [andrewd18/df-lnp-installer#6]
An error window pops up: 

Image not found
Not found: data/art/mouse.png

**Comment by andrewd18** [comment]
Confirmed on 0.1.3.

**Workaround:**
From your df folder, run:

```
 cp ./LNP/graphics/ASCII\ Default/data/art/mouse.png ./df_linux/data/art/
```

**Comment by andrewd18** [comment]
Resolved in 0.1.4. Either pull and reinstall or use the workaround. Thanks for the bug report!
